group_vars/dnbd3/vars.yml

```diff
@@ -11,5 +12,5 @@ dnbd3_primary_comment: "Primary DNBD3 server, which replicates S3FS mounted QCOW
 # dnbd3_primary_backup: "10.20.56.174"
 # dnbd3_primary_backup_comment: "Secondary/backup DNBD3 server, which replicates S3FS mounted QCOW2 images."
 dnbd3_poxys:
-  - ip: "{{ hostvars['dnbd3-proxy.galaxyproject.eu']['ansible_default_ipv4']['address'] }}"
+  - ip: "{{ hostvars['sn12.galaxyproject.eu']['ansible_default_ipv4']['address'] }}"
```

Comment:
Is that change correct? I thought you wanted to have the more specific names?

Reply:
The idea was to use host-based variables for this machine, because we should only have one place where we configure it's firewall and that should go to the host-vars.
this is why I am referring to it as sn12. For the dnbd3 service it does not matter, because as visible here, it uses IP anyway.
The `dnbd3-proxy.galaxyproject.eu` CNAME should be added to infrastructure anyway